scripts/index.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+# This script will generate index for official stacks
+
+set -e
+
+OFFICIAL_STACKS=$(find -s ./official-stack ! -name 'cue.mod' -maxdepth 1 -mindepth 1 -type d | cut -d ' ' -f 1)
```

Comment:
I recommend handle these tasks in `scripts/process_stacks.sh` file.

Just wrap `generate_index()` as another function to be called.